frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -63,19 +66,56 @@ const Campaigns = () => {
     setLoading(false)
   }
 
+  // Returns true if lastSeenVersion < currentVersion, else false
+  function compareSemver(currentVersion: string, lastSeenVersion: string) {
```

Comment:
Minor nitpick. Shouldn't currentVersion and lastSeenVersion be flipped? The function is called below with `compareSemver(announcementVersion, i18n._(ANNOUNCEMENT.version))`